.changeset/kind-pets-sin.md

```diff
@@ -0,0 +1,15 @@
+---
+'@firebase/app': minor
+'firebase': minor
+'@firebase/data-connect': patch
+'@firebase/firestore': patch
+'@firebase/functions': patch
+'@firebase/database': patch
+'@firebase/vertexai': patch
+'@firebase/storage': patch
+'@firebase/auth': patch
+---
+
+`FirebaseServerApp` may now be initalized with an App Check token in leu of invoking the App Check
```

Comment:
Do these strings get pulled into release notes or other public docs? 

If so, I'd go with "can now be initialized" and "instead of invoking."

Reply:
They don't directly. Later, the release engineer crafts the release notes using these as ... inspiration. Updated!